src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -81,6 +97,16 @@ public ArrayList<Integer> getFavouriteNumbers() {
          - A string
          The method must remove the second parameter from the list and then return the list
      */
+    public ArrayList<String> removeIngredient(ArrayList<String> list, String text){
+        for(int i = 0; i < list.size(); i++){
+            if(list.get(i).equals(text)){
+                list.remove(i);
+
+            }
+
+        }
+        return list;
+    }
```

Comment:
Could be simplified, eg:

public ArrayList<String> removeIngredient(ArrayList<String> list, String text) {
    list.remove(text);
    return list;
}